core/assets/bundles/bundle_fr.properties

```
@@ -354,7 +354,7 @@ command.rebuild = Reconstruire
 command.assist = Assister
 command.move = Bouger
 command.boost = Booster
-command.loadUnits = Transporter Unités
+command.loadUnits = Transporter Unitées
```

```suggestion
command.loadUnits = Transporter Unités
```
Ce n'est pas correct. "Unité" n'a pas de "e", même si c'est féminin.